image/Image/Image.css

```diff
@@ -0,0 +1,31 @@
+@import "../../utility/component.css";
+
+.image-size-extrasmall{
+    width: 20px;
+    height: 20px;
+}
+
+.image-size-small{
+    width: 50px;
+    height: 50px;
+}
+
+.image-size-medium{
+    width: 100px;
+    height: 100px;
+}
+
+.image-size-larger{
```

Comment:
Please try to use relative units everywhere..If you have time .. else you can do it after 16